freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -284,6 +289,13 @@ public int getUsernameLength() throws FreeIpaClientException {
         return config.getIpamaxusernamelength();
     }
 
+    public Config getConfig() throws FreeIpaClientException {
+        Map<String, Object> params = new HashMap<>();
+        params.put("all", Boolean.TRUE);
+        Config config = (Config) invoke("config_show", List.of(), params, Config.class).getResult();
+        return config;
```

Comment:
no need for this extra line, just put return above

Reply:
Done, this was for my breakpoint.